tests/integration/llm/test_deprecated_providers.py

```
@@ -0,0 +1,417 @@
+    - Verify comprehensive 4-component error message
+    """
+
+    def test_quickstart_v1_gemini_rejection_scenario(self):
+        """
+        Complete V1 scenario: Deprecated provider rejection with comprehensive error.
+
+        This test exactly mirrors the validation steps in quickstart.md V1.
+        """
+        # Step 1: Attempt to create template with "gemini"
+        with pytest.raises(ValueError) as exc_info:
+            template = ReportTemplate(
```

Ruff (0.14.1)

Local variable `template` is assigned to but never used

Remove assignment to unused variable `template`

(F841)